src/components/pipeline-runs/pipeline-run-table-row.tsx

```diff
@@ -0,0 +1,68 @@
+import { Table, Typography } from '@equinor/eds-core-react';
+import * as PropTypes from 'prop-types';
+
+import { StatusBadge } from '../status-badges';
+import { Duration } from '../time/duration';
+import { RelativeToNow } from '../time/relative-to-now';
+import {
+  PipelineRunModel,
+  PipelineRunModelValidationMap,
+} from '../../models/pipeline-run';
+import { Link } from 'react-router-dom';
+import { routeWithParams } from '../../utils/string';
+import routes from '../../routes';
+export interface PipelineRunTableRowProps {
```

Comment:
add gap between import and declaration
```suggestion
import routes from '../../routes';

export interface PipelineRunTableRowProps {
```